camel/models/model_factory.py

```diff
@@ -46,6 +46,7 @@ def create(model_type: ModelType,
                 ModelType.GPT_3_5_TURBO_16K,
                 ModelType.GPT_4,
                 ModelType.GPT_4_32k,
+                ModelType.TEXT_DAVINCI_003,
```

Comment:
Why do we need to add a deprecated model?
I believe `ModelType.GPT_3_5_TURBO` can do simple completion as well (not only chat completion).

Reply:
As I have chosen to use the `Completion` API, the model is related to `TEXT_DAVINCI_003`, and its quality of response is much better than `TURBO` in the case of filling in the blanks. And `Completion` is not supported by the `TURBO` model.

Reply:
Now I have removed the model according to your comment.